QUERIES TASK.sql

```diff
@@ -0,0 +1,120 @@
+--1- every customer (E-Shop) 
+--wants to know who the top 3 customers buy from them
+SELECT c.Name as TOP_3_CUSTOMERS
```

Comment:
I mean by the top 3 customers who have more orders
example:
#1 = Ahmad, he ordered 10 times
#2 = Ali, he ordered 6times
#3 = Mai, she ordered 5 times

also, try the keyword `top`